src/server/verification/faceRecognition/faceRecognitionHelper.js

```diff
@@ -121,7 +129,8 @@ const Helper = {
   async delete(enrollmentIdentifier: string): Promise<boolean> {
     try {
       let res = await ZoomClient.delete(enrollmentIdentifier)
-      log.debug('delete result:', { res })
+      let results = res.meta.ok
+      results && log.debug('delete result:', { results })
```

Comment:
same

Reply:
removed, commit ed4ee0da4d09f3e67a4e8523866aaeddd36ed7b2, will be resolved once pushed